### PR TITLE
[FIX] Popover position for arabic languages

### LINKED ISSUE #23599: Avatar overlap on user name in RTL
### Description:
In a chat, the user Avatar overlap the user name and first message sent

### Steps to reproduce:

1. Open RC in RTL languagr
2. Select another user to send a message

### Expected behavior:

User avatar does not overlap and hid user name and first message
### Actual behavior:

User avatar overlap and hid user name and first message
### Server Setup Information:

- Version of Rocket.Chat Server: 4..1.0
- Operating System: ubuntu 18.04 LTS
- Deployment Method:  manual installation
- Number of Running Instances: 1
- DB Replicaset Oplog: wiredTiger (oplog Enabled)
- NodeJS Version: v12.22.1
- MongoDB Version:4.2.17 

### Client Setup Information

- Desktop App or Browser Version: FF
- Operating System: ubuntu

### Relevant logs:

Screenshot 2021-10-31 at 00-44-33 مِـرســال

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Issue(s)

Closes #23599

## Steps to test or reproduce

- change the RC language to some arabic language.
- Userdrop down goes out of the screen on rightside

### CODE REVIEW

File: app/ui-utils/client/lib/popover.js [hunk 1]
```diff
@@ -143,6 +148,7 @@ Template.popover.onDestroyed(function() {
 		this.data.onDestroyed();
 	}
 	$(window).off('resize', this.position);
+	this.observer && this.observer.disconnect();
```
Comment (dougfabris) [inline app/ui-utils/client/lib/popover.js:151]:
```suggestion
	this.observer?.disconnect();
```

**Review by dougfabris — CHANGES_REQUESTED** [review]

**Review by dougfabris — APPROVED** [review]

**Comment by ali-alhaidary** [comment]
https://github.com/RocketChat/Rocket.Chat/issues/23599

**Comment by ali-alhaidary** [comment]
https://github.com/RocketChat/Rocket.Chat/issues/22923

**Comment by ali-alhaidary** [comment]
Still in v4.3.0

**Comment by dougfabris** [comment]
> Still in v4.3.0

Can you provide more details? It's working fine here: 
image

**Comment by ali-alhaidary** [comment]
> > Still in v4.3.0
> 
> @ali-alhaidary Can you provide more details? It's working fine here: image

https://github.com/RocketChat/Rocket.Chat/issues/23599

still in 4.3.1

**Comment by dougfabris** [comment]
@ali-alhaidary Please, provide some screenshots or logs. We couldn't reproduce it anymore!!

**Comment by JeroenAdam** [comment]
I can confirm this is solved. Upgraded from 4.2.2 to 4.3.1.

**Comment by ali-alhaidary** [comment]
https://github.com/RocketChat/Rocket.Chat/issues/23599
Still in v4.3.2

**Comment by dougfabris** [comment]
@ali-alhaidary As I'm trying to say to you I'm not able to reproduce the error. Just sending us that the issue remains won't help us at all. Please, provide more details!
The screenshot above is an attempt to reproduce. Am I missing something?
image

**Comment by ali-alhaidary** [comment]
still in 4.5.1

**Comment by ali-alhaidary** [comment]
still in 4.4.2

**Pull request merged**